opentelemetry-sdk/CHANGELOG.md

```diff
@@ -2,25 +2,42 @@
 
-- *Bug fix*: ObservableGauge returns data points recorded since previous collection, despite temporality. Other asynchronous (observable) instruments with Cumulative temporality behave as synchronous ones and return data points on every collection. [#2213](https://github.com/open-telemetry/opentelemetry-rust/issues/2213)
-
-- *Feature*: Introduced a new feature flag, `experimental_metrics_disable_name_validation`, under the `opentelemetry-sdk`, which allows disabling the Instrument Name Validation. This is useful in scenarios where you need to use *special characters*, *Windows Perf Counter Wildcard Path*, or similar cases. For more details, check [#2543](https://github.com/open-telemetry/opentelemetry-rust/pull/2543).
-  > **WARNING**: While this feature provides flexibility, **be cautious** when using it, as platforms like **Prometheus** impose restrictions on metric names and labels (e.g., no spaces, capital letters, or certain special characters). Using invalid characters may result in compatibility issues or incorrect behavior. Ensure that instrument names comply with the requirements of your target platform to avoid potential problems.
-
-- *Breaking(Affects custom metric exporter authors only)* `start_time` and `time` is moved from DataPoints to aggregations (Sum, Gauge, Histogram, ExpoHistogram) see [#2377](https://github.com/open-telemetry/opentelemetry-rust/pull/2377) and [#2411](https://github.com/open-telemetry/opentelemetry-rust/pull/2411), to reduce memory.
-
-- *Breaking* `start_time` is no longer optional for `Sum` aggregation, see [#2367](https://github.com/open-telemetry/opentelemetry-rust/pull/2367), but is still optional for `Gauge` aggregation see [#2389](https://github.com/open-telemetry/opentelemetry-rust/pull/2389).
-
-- *Breaking*
-  - SimpleLogProcessor modified to be generic over `LogExporter` to
-    avoid dynamic dispatch to invoke exporter. If you were using
-    `with_simple_exporter` to add `LogExporter` with SimpleLogProcessor, this is a
-    transparent change.
-    [#2338](https://github.com/open-telemetry/opentelemetry-rust/pull/2338)
-  - `ResourceDetector.detect()` no longer supports timeout option.
-  - `opentelemetry::global::shutdown_tracer_provider()` removed from the API, should now use `tracer_provider.shutdown()` see [#2369](https://github.com/open-telemetry/opentelemetry-rust/pull/2369) for a migration example. "Tracer provider" is cheaply clonable, so users are encouraged to set a clone of it as the global (ex: `global::set_tracer_provider(provider.clone()))`, so that instrumentations and other components can obtain tracers from `global::tracer()`. The tracer_provider must be kept around to call shutdown on it at the end of application (ex: `tracer_provider.shutdown()`)
+Note: Due to the large amount of making changes, check [migration guide to
+0.28](../docs/migration_0.28.md) for a summary that can help majority users to
+quickly migrate. The changelog below is the full list of changes.
```

Comment:
Does it make sense to group these changes based on the signals, and exporters this time, would be easy to users to see the relevant part which affects them?

Reply:
most changes affect multiple signals... But still I can do another reordering to get signals specific one close to each other